.github/renovate.json5

```diff
@@ -23,9 +40,9 @@
       matchManagers: ["github-actions"],
     },
     {
-      groupName: "ubuntu",
+      groupName: "ubuntu & python upgrade",
       groupSlug: "ubuntu",
-      matchDepNames: ["ubuntu"],
+      matchDepNames: ["ubuntu", "python"],
       dependencyDashboardApproval: true, // manual update when deployment is updated
     },
```

Comment:
Dose this make it so that we get a suggestion for the current Python version of Ubuntu, or the latest Python version? I would expect the latter. I think then we should just drop Python updates, because the latest Python version might still be newer, and then this change blocks the Ubuntu update we actually want

Reply:
The latter.
You are right, although I'd say that the python version will be updated in a new ubuntu version almost certainly, so we have to edit the version anyway. I thought that there may be a non-zero chance that the updated python version corresponds to the ubuntu python version and this won't increase the workload.
If we don't change the python in the ubuntu update, we may forget updating the supported python version in the toml file, making it unusable on that ubuntu version. The error caused by the update may remind us of that.
I also see ignoring python or even disabling both. Renovate won't be able to create a functional PR directly. That's why we initially picked manual PR creation for ubuntu.

Reply:
My vote is to handle python the same as ubuntu: Make it show up in the Dashboard-Issue, but don't auto-open PRs for it, we handle it manually (and likely do not use Renovate-PR for this, but do everything manually)

If I understand correctly, this is exactly what the change here does, then 👍 from me on the current version here.

Reply:
Doesn't the current change group the ubuntu and Python updates together in one PR?

I would be fine with Richard's approach, which I understand as having one PR for ubuntu and one for Python, but both manually triggered

Reply:
Yes, this change here groups them. So far we've kept Ubuntu and Python in sync, so this makes sense to me, but I'd also be ok to have two separate checkboxes. My understanding is that the Renovate-created PR is mostly useless for these anyway, so personally, I wouldn't click the checkbox, but create the PR manually.

Reply:
Okay, either way is fine